src/test/kotlin/lotto/model/LottoNumbersTest.kt

```diff
@@ -0,0 +1,39 @@
+package lotto.model
+
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.Arguments
+import org.junit.jupiter.params.provider.MethodSource
+
+internal class LottoNumbersTest {
+    @ParameterizedTest
+    @MethodSource("numbersProvider")
+    fun `로또 숫자는 6개로 이루어져 있다`(numbers: List<Int>) {
+        assertThrows<IllegalArgumentException> {
+            LottoNumbers(numbers)
+        }
+    }
+
+    @Test
+    fun `로또 숫자는 출력될때 정렬 되어야 한다`() {
+        assertThrows<java.lang.IllegalArgumentException> {
```

Comment:
코틀린의 `IllegalArgumentException`을 사용하면 어떨까요?

```suggestion
        assertThrows<IllegalArgumentException> {
```